src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -16,9 +16,36 @@
 import org.kohsuke.args4j.CmdLineException;
 import org.kohsuke.args4j.CmdLineParser;
 import org.kohsuke.args4j.Option;
-import org.kohsuke.args4j.spi.StringArrayOptionHandler;
+import org.kohsuke.args4j.OptionDef;
+import org.kohsuke.args4j.spi.OptionHandler;
+import org.kohsuke.args4j.spi.Parameters;
+import org.kohsuke.args4j.spi.Setter;
 
 public class Options {
+  public static class StringArrayOptionHandler extends OptionHandler<String> {
```

Comment:
maybe this should have some minimal docs explaining why we have our own handler here (in particular given it has the same name as the standard one)?

Reply:
Sure thing, I'll add a docblock